element-connector/src/main/java/org/eclipse/californium/elements/tcp/TcpClientConnector.java

```diff
@@ -59,10 +68,12 @@ public TcpClientConnector(int numberOfThreads, int connectTimeoutMillis, int idl
 		this.numberOfThreads = numberOfThreads;
 		this.connectionIdleTimeoutSeconds = idleTimeout;
 		this.connectTimeoutMillis = connectTimeoutMillis;
-		this.listenUri = URI.create(String.format("%s://127.0.0.1:0", getSupportedScheme()));
+		this.listenUri = URI.create(String.format("%s://%s:%d", getSupportedScheme(),
```

Comment:
I really think, that "127.0.0.1" is missleading. Using that will result in "0.0.0.0".